CMakeLists.txt

```diff
@@ -331,6 +305,80 @@ add_swift_library(Foundation
                     $<$<PLATFORM_ID:Windows>:CoreFoundationResources>)
 
+add_swift_library(FoundationNetworking
+                  MODULE_NAME
+                    FoundationNetworking
+                  MODULE_LINK_NAME
+                    FoundationNetworking
+                  MODULE_PATH
+                    ${CMAKE_CURRENT_BINARY_DIR}/swift/FoundationNetworking.swiftmodule
+                  SOURCES
+                  Foundation/Boxing.swift
+                  Foundation/NSURLRequest.swift
+                  Foundation/HTTPCookie.swift
+                  Foundation/HTTPCookieStorage.swift
+                  Foundation/URLAuthenticationChallenge.swift
+                  Foundation/URLCache.swift
+                  Foundation/URLCredential.swift
+                  Foundation/URLCredentialStorage.swift
+                  Foundation/URLProtectionSpace.swift
+                  Foundation/URLProtocol.swift
+                  Foundation/URLRequest.swift
+                  Foundation/URLResponse.swift
+                  Foundation/URLSession/BodySource.swift
+                  Foundation/URLSession/Configuration.swift
+                  Foundation/URLSession/http/HTTPMessage.swift
+                  Foundation/URLSession/http/HTTPURLProtocol.swift
+                  Foundation/URLSession/libcurl/EasyHandle.swift
+                  Foundation/URLSession/libcurl/libcurlHelpers.swift
+                  Foundation/URLSession/libcurl/MultiHandle.swift
+                  Foundation/URLSession/Message.swift
+                  Foundation/URLSession/NativeProtocol.swift
+                  Foundation/URLSession/NetworkingSpecific.swift
+                  Foundation/URLSession/ftp/FTPURLProtocol.swift
+                  Foundation/URLSession/TaskRegistry.swift
+                  Foundation/URLSession/TransferState.swift
+                  Foundation/URLSession/URLSession.swift
+                  Foundation/URLSession/URLSessionConfiguration.swift
+                  Foundation/URLSession/URLSessionDelegate.swift
+                  Foundation/URLSession/URLSessionTask.swift
+                  TARGET
+                    ${CMAKE_C_COMPILER_TARGET}
+                  CFLAGS
+                    ${MSVCRT_C_FLAGS}
+                    ${deployment_target}
+                    ${deployment_enable_libdispatch}
+                    -F${CMAKE_CURRENT_BINARY_DIR}
+                  LINK_FLAGS
+                    ${MSVCRT_LINK_FLAGS}
+                    -L${CMAKE_CURRENT_BINARY_DIR}
+                    ${ICU_UC_LIBRARY} ${ICU_I18N_LIBRARY}
+                    ${libdispatch_ldflags}
+                    -lFoundation
+                    ${Foundation_INTERFACE_LIBRARIES}
+					${CFURLSessionInterface_LIBRARIES}
+                    ${CURL_LIBRARIES}
+                    ${Foundation_RPATH}
+                    ${WORKAROUND_SR9138}
+                    ${WORKAROUND_SR9995}
+                    $<$<PLATFORM_ID:Windows>:$<TARGET_OBJECTS:CoreFoundationResources>>
```

Comment:
Why do you need the CoreFoundationResources?  That is for the TimeZone information